workers/data_refinery_workers/processors/salmon.py

```diff
@@ -219,7 +290,7 @@ def _run_fastqc(job_context: Dict) -> Dict:
 
     # We could use --noextract here, but MultiQC wants extracted files.
     command_str = ("./FastQC/fastqc --outdir={qc_directory} {files}")
-    files = ' '.join(file.absolute_file_path for file in job_context['original_files'])
+    files = ' '.join([job_context.get('input_file_path_2', ''), job_context.get('input_file_path_2', '')])
```

Comment:
I would think that one of these would be `input_file_path` instead of `input_file_path_2` twice.